cloud/scope/cluster_test.go

```diff
@@ -114,7 +114,14 @@ func TestClusterScopeMethods(t *testing.T) {
 					},
 				},
 			},
-			patchErr: nil,
+			expects: func(mock *mock.Mockk8sClient) {
+				mock.EXPECT().Scheme().DoAndReturn(func() *runtime.Scheme {
+					s := runtime.NewScheme()
+					infrav1alpha1.AddToScheme(s)
+					return s
+				}).AnyTimes()
```

Comment:
```suggestion
				})
```
`AnyTimes` means that it can also be called 0 times.